installer.sh

```diff
@@ -545,26 +552,40 @@ create_user_freebsd() {
 		tty
 	)
 	for grp in "${groups[@]}"; do
-		if [ "$IS_ROOT" = true ]; then
-			getent group $grp && pw group mod $grp -m $username
-		else
-			getent group $grp && sudo pw group mod $grp -m $username
-		fi
+		getent group $grp && $SUDOX pw group mod $grp -m $username
 	done
 }
 
+fix_dir_permissions() {
+	# When autostart is enabled, we need to fix the permissions so that `iobroker` can access it
+	echo "Fixing directory permissions..."
+	$SUDOX chown -R $IOB_USER:$IOB_USER $IOB_DIR
+	if [ "$IS_ROOT" != true ]; then
+		# To allow the current user to install adapters via the shell,
+		# We need to give it access rights to the directory aswell
```

Comment:
I think this comment is outdated now. This allows the user to edit and view files on the shell. Installation is now patched as we discussed earlier.

Reply:
Deleted